HIP/hip-412.md

```diff
@@ -360,67 +522,110 @@ The following is the formal definition of this schema using JSON Schema notation
 For more info see here: https://json-schema.org/
 ```json
 {
```

Comment:
The GIST created for the schema needs updating to reflect what's here as it's out of date currently. Once finalised we could upload the schema definition to IPFS so we have a dedicated link to it for everyone to use.

Reply:
Should be reflected now: https://gist.github.com/michielmulders/571c496789ede04c9074817cee834246